docs/cog_guides/alias.rst

```diff
@@ -0,0 +1,270 @@
+.. _alias:
+
+=====
+Alias
+=====
+
+This is the cog guide for the alias cog. You will
+find detailed docs about the usage and the commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load alias
+
+.. _alias-usage:
+
+-----
+Usage
```

Comment:
If you are going to start off by showing off examples of aliases (without the users necessarily knowing what any of the commands are) you probably do not want to use commands that have parameters. Also, the explanation should probably be with each command, not in a chunk after the two.